internal/tools/clickhouse_create_blocks_table.sql

```diff
@@ -1,4 +1,4 @@
-CREATE TABLE base.blocks (
+CREATE TABLE blocks (
```

Comment:
this means data from different chains will be stored in one table?

Reply:
We have chain ID in every table, so we can put them all in one